torch_xla/csrc/xla_sharding_util.cpp

```diff
@@ -86,6 +86,65 @@ xla::HloModuleProto ShardingUtil::SpmdPartitioningPass(
   return module.get()->ToProto();
 }
 
+std::vector<std::vector<xla::ComputationClient::DataPtr>>
+ShardingUtil::InputHandler(
```

Comment:
can you add a comment to this function? Name doesn't tell me much regarding how I should use this function

Reply:
nvm, say it in .h